src/components/Home.jsx

```diff
@@ -1,7 +1,44 @@
-import { Link } from "react-router-dom";
-
 const Home = ({ articles }) => {
-  return <main id="homeMain">test</main>;
+  return (
+    <main id="homeMain">
+      {articles.map((article) => {
+        return (
+          <ul id="article">
```

Comment:
if we have the ul outside of the map then we can have all the elements inside one li :) this would then make each article a li in the list rather than each element in the article a list